var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -19,6 +17,7 @@ class Podio(CMakePackage):
     tags = ["hep", "key4hep"]
 
     version('master', branch='master')
+    version('0.13', sha256='e9cbd4e25730003d3706ad82e28b15cb5bdc524a78b0a26e90b89ea852101498')
```

Comment:
Can we call this `0.13.0`? In this way we'll be consistent with the scheme of the other versions. Note that upstream don't use this numbering scheme, they instead use a weird `00-13`, `00-12`, ...

Reply:
There's a good reason to call this `0.13`: it will compare equal to the remote version `Version('0.13') == Version('00-13')`, but `Version('0.13.0') != Version('00-13')` (so it works for example with the new `spack versions --new podio`). I should have done this from the beginning, but I can actually change the other versions to use this format as well.

Agreed that the upstream numbering scheme is not ideal, but that's been applied to a lot of packages in hep ...

Reply:
> I should have done this from the beginning, but I can actually change the other versions to use this format as well.

If that is not an issue with the HEP community, I would do it.

Reply:
Ok done! I don't think it's an issue.